rdrf/rdrf/models/definition/models.py

```diff
@@ -633,6 +633,12 @@ def proms_system_url(self):
         except KeyError:
             return None
 
+    def registration_allowed(self):
+        registration_enabled = self.has_feature(RegistryFeatures.REGISTRATION)
+        return (
+            registration_enabled and EmailNotification.objects.filter(registry=self, disabled=False).exists()
```

Comment:
Can we be more specific here?
There are multiple types of email notifications, we would like to check that we have user registration notifications set up.

Reply:
Updated.